### DISCUSSION TITLE
object-oriented.ts gone?

### DISCUSSION THREAD

**Discussion by shaneafsar** [neet/masto.js#774] [Q&A] [answered]
Just curious -- if my eyes don't deceive me, why was `examples/object-oriented.ts` removed? Loved that example!

**Comment by neet** [comment, marked as answer]
Haha, I didn't expect anybody who notice it. I remove the OOP example because...

- I wanted to turn the examples into a definitive explanation of how to use Masto.js, not a general coding technique
- It uses `class`, but is not quite object-oriented. I thought you have to use other techniques like encapsulation, delegation, or SOLID principle when you name it OOP
- It has a duplicated usage as other examples and I don't find a necessity for putting it as an independent guide.

I'll keep trying to find a way for providing more helpful information on using Masto.js 🙇

**Reply by shaneafsar** [reply]
Fair enough, thank you for the explanation! I was just curious because I found the example to be super helpful to reference while moving a twitter bot over to mastodon (matched similar patterns).

**Reply by shaneafsar** [reply]
And thank for this awesome library!
